inference-engine/tests/functional/inference_engine/serialization/single_layer/convert.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/convert.hpp"
+
+#include <vector>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+const std::vector<std::vector<size_t>> inShape = {{1, 2, 3, 4}};
+
+const std::vector<InferenceEngine::Precision> precisions = {
+    InferenceEngine::Precision::BOOL, InferenceEngine::Precision::U8,
```

Comment:
Can we add serialisation tests for lp types (u1, u4, i4)?

Reply:
Yes, but in next PR, at current state they will fail because of new checks in validate_and_infer_types (they are going to be removed in LP types PR).